cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -44,8 +44,12 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         |These administrative commands are automatically routed to the `system` database when connected to the DBMS over Bolt.""".stripMargin)
     p("include::databases-command-syntax.asciidoc[]")
     section("Listing databases", "administration-databases-show-databases") {
-      p("There are three different commands for listing databases. Listing all databases, listing a particular database or listing the default database.")
-      p("All available databases can be seen using the command `SHOW DATABASES`.")
+      p(
+        """There are three different commands for listing databases. Listing all databases, listing a particular database or listing the default database.
+          |The `SHOW DATABASES` commands return the following columns:""".stripMargin)
```

Comment:
```suggestion
          |These commands return the following columns:""".stripMargin)
```
since they were just listed and that syntax does not fit all of the commands?